app/controllers/web_api_controller.rb

```diff
@@ -0,0 +1,12 @@
+# This controller provides UI access to investigate and
+# browse the RESTful API endpoints using a Swagger UI
+# and the automapping from the Swagger data at /api/swagger.json
+#
+# Swagger UI migrated from original repository at https://github.com/swagger-api/swagger-ui
+
+class WebApiController < ApplicationController
```

Comment:
The reason it's a controller method is that, in the future, we'll probably have a dynamic list of RESTful endpoints.  The UI itself is built to talk to any swagger-compliant descriptors, but the default implementation is that the user loads up the page and types in the URL to the endpoint.  

In our case, we'll most likely end up with a set of services from different projects.  Adding or configuring a dynamic page to access those starts with having a dynamic controller.